src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1635,7 +1647,7 @@ private void ToggleTreeViewItemHighlighting(string path, bool isSelected)
         /// attaches them to the visual scene.
         /// </summary>
         /// <param name="packages">An <see cref="IEnumerable"/> of <see cref="HelixRenderPackage"/>.</param>
-        private void AggregateRenderPackages(IEnumerable<HelixRenderPackage> packages)
+        protected virtual void AggregateRenderPackages(IEnumerable<HelixRenderPackage> packages)
```

Comment:
This has the effect of exposing this method to the API in any derived class, as this is currently a public method - (In the current API) Of course we'd like to do a refactor and change our API policies as we have been discussing but for now I'm a bit hesitant.

If this method was made internal could it be mocked?

If it has to become protected then we can use an interface as a the argument type and check the type inside the method.

Reply:
> If this method was made internal could it be mocked?

Apparently this could be done https://stackoverflow.com/a/2823554. Let me give that a try.

Reply:
@mjkkirschner That worked! Would you take one last look?